src/test/index.test.ts

```diff
@@ -1,20 +1,11 @@
 import type { Express } from 'express'
-import type { MetricsConfigurationOptions } from '../index'
 
 const request = require('supertest')
 const express = require('express')
 const path = require('path')
 const fs = require('fs')
 let { env } = require('node:process')
 
-const metricsConfig: MetricsConfigurationOptions = {
-  defaultMetricsLabels: {
-    one: 'test1',
-    two: 'test2',
-    three: 'test3',
-  },
-}
-
 jest.mock('node:http', () => ({
```

Comment:
this mock can be removed now that the client doesn't make any http requests

Reply:
👍